super/Login_db/bot.py

```
@@ -69,8 +69,33 @@ def jls_extract_def():
 
 main_args, credentials = jls_extract_def()
 
-def log_one(site, user, result, action=""):
+userinfo = 0
 
+def log_one(site, user, result, action="", params={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)